docs/operation/property-repair.md

```diff
@@ -0,0 +1,60 @@
+# Property Background Repair Observability
+
+Based on the [Property Background Repair Strategy documentation](../concept/property-repair.md), 
+this article explains how to visualize and monitor each synchronization cycle to enhance observability and debugging.
+
+This feature is enabled by default. You can configure whether to record the data through the `--property-repair-obs-enabled` option.
+
+## Tracing
+
+Tracing is used to record the operation flow at each node during gossip-based Property repair. 
+This allows for fast and accurate diagnosis in the event of issues or inconsistencies during the repair process.
+
+All trace data is written to the `_property_repair_spans` stream group and can be queried for inspection and analysis.
+
+| Field Name        | Type     | Description                                                                          |
+|-------------------|----------|--------------------------------------------------------------------------------------|
+| `tracing_id`      | `string` | Unique trace ID for the entire repair task, typically `sender_node_id + start_time`. |
```

Comment:
Add `sender_node_id` and set it as the [`entity`](https://github.com/apache/skywalking-banyandb/blob/main/api/proto/banyandb/database/v1/schema.proto#L59)